File: app/src/main/java/com/lumiyaviewer/lumiya/slproto/modules/rlv/commands/RLVCmdAcceptTeleport.java
package com.linkpoint.slproto.modules.rlv.commands;

import com.linkpoint.slproto.modules.rlv.RLVRestrictionType;

public class RLVCmdAcceptTeleport extends RLVCmdGenericRestriction {
    public RLVCmdAcceptTeleport() {
        super(RLVRestrictionType.accepttp, true);
    }
}

File: Linkpoint/src/main/kotlin/com/linkpoint/slproto/modules/rlv/commands/RLVCmdAcceptTeleport.kt
package com.linkpoint.slproto.modules.rlv.commands

import com.linkpoint.slproto.modules.rlv.RLVRestrictionType

class RLVCmdAcceptTeleport : RLVCmdGenericRestriction() {
    public RLVCmdAcceptTeleport() {
        super(RLVRestrictionType.accepttp, true)
    }
}
